hardware/measurement/test_adc_stream_rate.py

```
@@ -0,0 +1,395 @@
+        for k, b in enumerate(blocks):
+            b.t_recv = k * 0.1
+        r = asr.summarize(blocks, p.stats, 0.0, 0.2, discard_blocks=0)
+        self.assertEqual(r["lost_blocks"], 1)
+
+        with tempfile.TemporaryDirectory() as d:
+            path = os.path.join(d, "out.csv")
+            # 欠落があるので既定では書かない。
+            with self.assertRaises(SystemExit):
+                asr.write_csv(path, r, allow_drops=False)
+            asr.write_csv(path, r, allow_drops=True)
+            rows = [l.strip().split(",") for l in
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)